### PR TITLE
add routing

### PR DESCRIPTION
**Pull request by Anita77777**

### CODE REVIEW

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+			</Link>
+
+			<Link passHref href="/tbr">
+				<button type="button" variant="nav">
+					<MySVG variant="bookFilled" />
+				</button>
+			</Link>
+		</footer>
+	);
+}
```
Comment (irenepapaspyratos) [inline src/components/Navbar/Navbar.js:38]: As I understood, a footer is normally used to display final information at the end of a page (foot...below..).
So first, putting a footer at the top is strange and second there should be info like impressum etc. (things, you don't want to see always but on scrollToEnd).
I think, what you want is a \<nav> . Maybe wrapped in a \<header>, but definitive a \<nav>.

**Review by irenepapaspyratos — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (Anita77777) [reply src/components/Navbar/Navbar.js:38]: Yeah the footer will be at the bottom as soon I style the page, next step. If you add a card the footer sludes down, but as I said the stylinf of the whole app is my next step after this PR

Comment (irenepapaspyratos) [reply src/components/Navbar/Navbar.js:38]: Ok, but...: nevertheless - top or bottom - what you have been building is a \<nav>, not a \<footer>. I strongly suggest to change that, as it is semantically wrong. Sorry...😔

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Link from 'next/link';
 import MySVG from '../SVG/SVG';
 import { useRouter } from 'next/router';
 
-export default function Navbar() {
+export default function Footer() {
```
Comment (irenepapaspyratos) [inline src/components/Footer/Footer.js]: Change back to Navbar

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import Link from 'next/link';
...
 	const router = useRouter();
 	return (
 		<footer>
```
Comment (irenepapaspyratos) [inline src/components/Footer/Footer.js]: Change \<footer> and \</footer> to \<nav> and \</nav>

File: src/components/Layout.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import Navbar from './Navbar/Navbar';
+import Footer from './Footer/Footer';
```
Comment (irenepapaspyratos) [inline src/components/Layout.js:2]: Change Footer back to Navbar

File: src/components/Layout.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 
 export default function Layout({ children }) {
 	return (
 		<>
-			<Navbar />
+			<Footer />
 			<main>{children}</main>
```
Comment (irenepapaspyratos) [inline src/components/Layout.js:7]: Change \<Footer /> back to \<Navbar />

**Review by irenepapaspyratos — CHANGES_REQUESTED** [review]

**Review by irenepapaspyratos — APPROVED** [review]

**Comment by irenepapaspyratos** [comment]
Very well done!! :)

**Pull request merged**